eq-author/src/App/page/Design/QuestionPageEditor/AnswersEditor/index.js

```diff
@@ -23,6 +23,10 @@ export const AnswersEditor = ({
   onDeleteAnswer,
   moveAnswer,
 }) => {
+
+    let multipleAnswers = false;
+    answers?.length > 1 ? multipleAnswers = true : multipleAnswers =false;
```

Comment:
```suggestion
    const multipleAnswers = answers?.length > 1 ? true : false;
```

Reply:
Ahh yes - nice and tidy - thanks!

Reply:
actually @MattElcock  this looks even cleaner!
const multipleAnswers = answers?.length > 1;
otherwise the linter jumps up and down!!!😁

Reply:
mmm - not so clever - now multipleAnswers won't exist as a required prop - I'm gonna settle on this...
let multipleAnswers = false;
multipleAnswers = answers?.length > 1;